skopt/space.py

```diff
@@ -42,6 +42,9 @@ class _CategoricalEncoder:
     def __init__(self):
         """Convert labeled categories into one-hot encoded features."""
         self._lb = LabelBinarizer()
+        NONE_ = '__None__'
+        self._Xin = np.vectorize(lambda a: a if a is not None else NONE_)
+        self._Xout = np.vectorize(lambda a: a if a is not NONE_ else None)
```

Comment:
rename it as `{en,de}code_none`? Something that is a bit more descriptive would be good.